.rubocop_todo.yml

```diff
@@ -59,15 +59,15 @@ Metrics/ClassLength:
     - 'lib/reports/cost_report.rb'
     - 'lib/cluster.rb'
 
-# Offense count: 6
+# Offense count: 7
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/MethodLength:
   Exclude:
     - 'bin/**/*.rb'
     - 'lib/scrub/autoscrub.rb'
     - 'lib/enum_chron_parser.rb'
     - 'lib/scrub/scrub_fields.rb'
-
+    - 'lib/clustering/cluster_ht_item.rb'
```

Comment:
Might be a good time to try `standardrb` instead of rubocop, which disables most of the complexity metrics.